src/exercises/exerciseTypes/audioExerciseTwo/AudioExerciseTwo.js

```diff
@@ -35,6 +36,8 @@ export default function AudioExerciseTwo({
   const [currentChoice, setCurrentChoice] = useState("");
   const [firstTypeTime, setFirstTypeTime] = useState();
   const [selectedButtonId, setSelectedButtonId] = useState("");
+  const bookmarkToStudy = bookmarksToStudy[0];
+   const exercise = "exercise";
```

Comment:
formatter needed again. 
install this pls:

<img width="850" alt="image" src="https://user-images.githubusercontent.com/464519/179272541-9c656512-e568-47e7-a2bb-fc5f5e15d841.png">